paasta_tools/utils.py

```diff
@@ -418,6 +419,10 @@ def get_instance(self) -> str:
     def get_service(self) -> str:
         return self.service
 
+    def get_namespace(self) -> str:
```

Comment:
We should be able to delete the version from KubernetesDeploymentConfig now that this is here, since it appears to be identical.